src/auth/SignUp.jsx

```diff
@@ -27,10 +28,16 @@ const initialNewUserState = {
 
 export const SignUp = () => {
   const [newUser, setNewUser] = useState(initialNewUserState);
+  const [error, setError] = useState(null);
+
```

Comment:
i know here i is the logical part, but we haven' t marked the required fields in the form yet did we?

Reply:
No, no yet, that's the browser on validation we can put it and the end, it makes write in every field.